talentsearch/webpack.mix.js

```diff
@@ -2,16 +2,20 @@ require('dotenv').config();
 require('laravel-mix-polyfill');
 
 const mix = require("laravel-mix");
+const path = require("path");
 
 let webpack = require('webpack')
 let dotenvplugin = new webpack.DefinePlugin({
   'process.env': {
-      API_URI: JSON.stringify(process.env.API_URI)
+      API_URI: JSON.stringify(process.env.API_URI),
+      TALENTSEARCH_APP_URL: JSON.stringify(process.env.TALENTSEARCH_APP_URL),
+      TALENTSEARCH_APP_DIR: JSON.stringify(process.env.TALENTSEARCH_APP_DIR),
```

Comment:
I see, so this is what's required to access env variables during the build process.